### PR TITLE
chore: update monorepo

### PR DESCRIPTION
**Pull request by johnxnguyen**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Update the monorepo from all `develop` branches of the old repos.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by SzymonMrozek — APPROVED** [review]

File: wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift [hunk 1]
```diff
@@ -29,6 +29,8 @@ class ZMUserSessionTests_Authentication: ZMUserSessionTestsBase {
         previousApiVersion = BackendInfo.apiVersion
         BackendInfo.apiVersion = .v0
 
+        BackendInfo.apiVersion = .v0
```
Comment (netbe) [inline wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift]: duplicate

File: wire-ios-sync-engine/Source/Calling/WireCallCenterV3+Events.swift [hunk 2]
```diff
@@ -203,6 +203,7 @@ extension WireCallCenterV3 {
                                            senderClientId: String,
                                            targets: AVSClientList?,
                                            data: Data) {
+        guard isEnabled else { return }
```
Comment (netbe) [inline wire-ios-sync-engine/Source/Calling/WireCallCenterV3+Events.swift]: duplicate

Comment (David-Henner) [inline wire-ios-sync-engine/Source/Calling/WireCallCenterV3+Events.swift]:
```suggestion
```

Comment (David-Henner) [reply wire-ios-sync-engine/Source/Calling/WireCallCenterV3+Events.swift]: looks like this is duplicated below

File: wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift [hunk 3]
```diff
@@ -29,6 +29,8 @@ class ZMUserSessionTests_Authentication: ZMUserSessionTestsBase {
...
+
```
Comment (David-Henner) [inline wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift]:
```suggestion
```

Comment (David-Henner) [reply wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift]: looks like a duplicate of line 30

File: wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift (hunk 1 above)
Comment (johnxnguyen) [inline wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift]:
```suggestion
```

Comment (johnxnguyen) [reply wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift]: fixed

File: wire-ios-sync-engine/Source/Calling/WireCallCenterV3+Events.swift (hunk 2 above)
Comment (johnxnguyen) [reply wire-ios-sync-engine/Source/Calling/WireCallCenterV3+Events.swift]: fixed

File: wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift (hunk 3 above)
Comment (johnxnguyen) [reply wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+Authentication.swift]: fixed

**Review by David-Henner — APPROVED** [review]

**Review by netbe — APPROVED** [review]

**Pull request merged**